mailer/urls.py

```diff
@@ -1,9 +1,7 @@
-try:
-    from django.conf.urls import patterns, url
-except ImportError:  # Django 1.4
-    from django.conf.urls.defaults import patterns, url
+from django.conf.urls import url
 
-urlpatterns = patterns(
-    'mailer.views',
-    url(r'^report/$', 'report', name='mailer_report'),
+import mailer.views
+
+urlpatterns = (
+    url(r'^report/$', mailer.views.report, name='mailer_report'),
```

Comment:
I thought it has to be a list, not a tuple now?  Or does it not matter for single urls

Reply:
It doesn't really matter unless you are adding static files urls in this file. But I'll change it.